apps/logo-quiz/src/app/views/LogoVerify/LogoVerify.tsx

```diff
@@ -235,39 +241,55 @@ class LogoVerify extends React.Component<LogoVerifyProps, LogoVerifyState> {
 
   render() {
     const options = this.props.options;
+    const isVerified = this.isVerified();
+
     return (
       <div className="logo-verify container">
         {this.props.logo && (
           <div className="header-wrapper">
             <Link to={`/levels/${this.props.logo.level}`} className="header-back">
-              <SVGBackArrow height="24px" />
+              <SVGBackArrow height="24px"/>
             </Link>
             <h3 className="header-title">Guess the logo!</h3>
           </div>
         )}
 
         {this.props.status === LogoStatus.Valid && (
           <div className="modal lv-modal">
-            <div className="modal__backdrop" />
+            <div className="modal__backdrop"/>
             <div className="modal__wrapper">
               <div className="modal__content lv-modal__content">
-                <SVGGreenCheckLg />
+                <SVGGreenCheckLg/>
                 <p>Good guess!</p>
                 {this.props.nextLogo && (
                   <Link
                     className="main__button lv-modal__button lv-modal__button--next"
                     to={this.props.nextLogo._id}
+                    innerRef={node => {
+                      // `node` refers to the mounted DOM element
+                      // or null when unmounted
+                      if (node) {
+                        node.focus();
+                      }
+                    }}
                   >
                     <span className="lv-modal__back-text">Next logo</span>
-                    <SVGBackArrow className="lv-modal__front-icon" height="16px" />
+                    <SVGBackArrow className="lv-modal__front-icon" height="16px"/>
                   </Link>
                 )}
-                <hr />
+                <hr/>
                 <Link
                   className="lv-modal__button lv-modal__button--prev"
                   to={`/levels/${this.props.logo.level}`}
+                  innerRef={node => {
+                    // `node` refers to the mounted DOM element
+                    // or null when unmounted
+                    if (node && isVerified) {
```

Comment:
Just wondering, why do you put focus on the "Back to logos" element when the logo is verified? In the demo, this is causing the focus to be placed in the back to logos instead of "next logo".

Reply:
sorry, my idea was to place the focus on the back to logos button, when the next logo button was hidden, but I messed up the condition, it is fixed now